perfkitbenchmarker/aws/aws_disk.py

```diff
@@ -56,7 +55,7 @@ def __init__(self, iops=None, **kwargs):
   def ApplyFlags(self, flags):
     """Apply flags to the DiskSpec."""
     super(AwsDiskSpec, self).ApplyFlags(flags)
-    self.iops = flags.scratch_disk_iops or self.iops
+    self.iops = flags.aws_provisioned_iops or self.iops
```

Comment:
Please retain existing flags at the first version of this work. 

We should only deprecate existing flags after new flags are checked-in and become well-understood and usable by the users.